packages/headless/src/index.ts

```diff
@@ -0,0 +1 @@
+export { createComboBox } from "./composables/createComboBox";
```

Comment:
```suggestion
export * from "./composables/createComboBox";
```

In case we export types etc. from the file in the future